### PR TITLE
files outside the docs now link the docs on docs.shopsys.com

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We have a new website (https://docs.shopsys.com/) where the documentation should be viewed primarly so it is sutiable to link the documentation on the site, not in github
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/read-model/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/shopsys/read-model.svg?branch=master)](https://travis-ci.org/shopsys/read-model)
 [![Downloads](https://img.shields.io/packagist/dt/shopsys/read-model.svg)](https://packagist.org/packages/shopsys/read-model)
 
-This bundle for [Shopsys Framework](https://www.shopsys.com) separates templates from model using [read model concept](https://github.com/shopsys/shopsys/blob/master/docs/model/introduction-to-read-model.md).
+This bundle for [Shopsys Framework](https://www.shopsys.com) separates templates from model using [read model concept](https://docs.shopsys.com/en/7.3/model/introduction-to-read-model).
```
Comment (grossmannmartin) [inline packages/read-model/README.md]:
```suggestion
This bundle for [Shopsys Framework](https://www.shopsys.com) separates templates from model using [read model concept](https://docs.shopsys.com/en/7.3/model/introduction-to-read-model/).
```

File: packages/read-model/src/Image/ImageView.php [hunk 2]
```diff
@@ -9,7 +9,7 @@
  *
  * Class representing images in frontend
  *
- * @see https://github.com/shopsys/shopsys/blob/master/docs/model/introduction-to-read-model.md
+ * @see https://docs.shopsys.com/en/7.3/model/introduction-to-read-model
```
Comment (grossmannmartin) [inline packages/read-model/src/Image/ImageView.php]:
```suggestion
 * @see https://docs.shopsys.com/en/7.3/model/introduction-to-read-model/
```

File: packages/read-model/src/Product/Listed/ListedProductView.php [hunk 3]
```diff
@@ -14,7 +14,7 @@
  *
  * Class representing products in lists in frontend
  *
- * @see https://github.com/shopsys/shopsys/blob/master/docs/model/introduction-to-read-model.md
+ * @see https://docs.shopsys.com/en/7.3/model/introduction-to-read-model
```
Comment (grossmannmartin) [inline packages/read-model/src/Product/Listed/ListedProductView.php]:
```suggestion
 * @see https://docs.shopsys.com/en/7.3/model/introduction-to-read-model/
```

**Comment by vitek-rostislav** [comment]
Hi, @grossmannmartin, thanks for the review :ok_hand: 

> What about Changelog file? Lines 863, 955, 994 and 1030 are without changed links on purpose, or it's a mistake?

This was on purpose - I was not sure in which version I should link the articles, the latest? :thinking: As those are old entries (for alpha versions) in the changelog, I do not mind keeping it as is, but I can change it if you want :slightly_smiling_face: 

> And I was a little confused with the whole contributing section. Do me it almost looks like articles that should not be versioned (and for example, BC Promise is really linked to latest version).

Actually, this is a good idea, I was not sure about that section too, always linking the files in contributing section to the latest versions makes sense to me, I will change it this way :wink:

**Pull request merged**